product-service/src/handlers/postProduct.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  APIGatewayProxyEvent,
+  APIGatewayProxyHandler,
+  APIGatewayProxyResult,
+} from 'aws-lambda';
+import 'source-map-support/register';
+import Joi from 'joi';
+import { Client } from 'pg';
+import dbOptions from '../dbConfig';
+
+console.log('DB CONFIG::', dbOptions);
+
+const postProduct = ` 
+  with ans1 as (
```

Comment:
Interesting syntax but it seems that Postgress actually wrappers any single statements in a transaction.
As with statement still count so there should be transaction under the hood.